prometheus_juju_exporter/config.py

```diff
@@ -0,0 +1,28 @@
+"""Configuration loader."""
+import confuse
+
+config = None
```

Comment:
Is there any reason to use global variable here?

Reply:
I use global config here to keep config consistent across the project. As you can see in the class init, we only call confuse.Configuration once no matter how many times we create Config() instance.

Reply:
Search `python` `singleton pattern` , you will find the answer to avoid global variable.

https://stackoverflow.com/questions/6760685/creating-a-singleton-in-python
https://mark1002.github.io/2018/07/31/python-%E5%AF%A6%E7%8F%BE-singleton-%E6%A8%A1%E5%BC%8F/

Maybe you already know `design pattern`, I just want to share it again for this cool knowledge.
https://refactoring.guru/design-patterns/behavioral-patterns